### ISSUE TITLE
Proposal: Simple task dependencies

### ISSUE THREAD

**Issue by geoffjentry** [ga4gh/task-execution-schemas#61]
There was a discussion on the mailing list about adding simple task dependency logic (i.e. this task can't run until that task has completed). Making an issue here for further discussion.

**Comment by buchanae** [comment]
Related mailing list discussions:
https://groups.google.com/a/genomicsandhealth.org/forum/#!topic/ga4gh-dwg-containers-workflows/qFq_jgoRCvs
https://groups.google.com/a/genomicsandhealth.org/forum/#!topic/ga4gh-dwg-containers-workflows/ccaqjysBvZY

**Comment by delagoya** [comment]
Google group name was changed. New URLs
https://groups.google.com/a/genomicsandhealth.org/forum/#!topic/ga4gh-cloud/qFq_jgoRCvs
https://groups.google.com/a/genomicsandhealth.org/forum/#!topic/ga4gh-cloud/ccaqjysBvZY

**Comment by uniqueg** [comment]
I think the way TES developed since 2017, I tend to be strongly against this (but I can't access the discussion to see pro arguments). I think what looks like "simple" now can easily become complex (what if it depends on multiple tasks? what if depends on tasks on _other_ TES instances? etc).

I vote for "won't fix".
